tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -120,25 +121,27 @@ public:
    /// TODO(jalopezg): at this point it would be nicer to distinguish between connecting a page sink and a page source
    void Connect(DescriptorId_t fieldId, RPageStorage *pageStorage, NTupleSize_t firstElementIndex = 0U);
 
-   void Append(const RColumnElementBase &element) {
+   void Append(const void *from)
+   {
       void *dst = fWritePage[fWritePageIdx].GrowUnchecked(1);
 
       if (fWritePage[fWritePageIdx].GetNElements() == fApproxNElementsPerPage / 2) {
          FlushShadowWritePage();
       }
 
-      element.WriteTo(dst, 1);
+      memcpy(dst, from, fElement->GetSize());
```

Comment:
Minor suggestion: also in line 162: I would stick either to `memcpy()` or `std::memcpy()` throughout the translation unit.